lib/common.js

```diff
@@ -75,6 +108,7 @@ var parseSetEnumTypeDef = function(type){
   }else if(type.substr(0,5).toLowerCase() === 'enum('){
     prefixLen = 5;
   }else{
+    // TODO: Remove throw statement to make module safe
```

Comment:
As a developer, I'm more interested in the frequency of parsing errors by type so I can try to isolate the problem and find commonalities between the occurrences. Perhaps accumulating these errors up until a threshold where a report can be written to disk or an event emitted would make the most sense. I think in the end, that's what people will want to implement themselves. That and a feature flag to throw on parsing errors so one can easily debug issues would be probably make everyone pretty happy. If you're hacking any lower-level than that you've already read most of the source code.